src/en/superstream/src/eu/kanade/tachiyomi/animeextension/en/superstream/SuperStreamAPI.kt

```diff
@@ -864,7 +864,7 @@ class SuperStreamAPI(val json: Json) {
     // Free Tibet, The Tienanmen Square protests of 1989
     private val iv = base64Decode("d0VpcGhUbiE=")
     private val key = base64Decode("MTIzZDZjZWRmNjI2ZHk1NDIzM2FhMXc2")
-    private val ip = base64Decode("aHR0cHM6Ly8xNTIuMzIuMTQ5LjE2MA==")
+    private val ip = base64Decode("aHR0cHM6Ly9zaG93Ym94LnNoZWd1Lm5ldA==")
```

Comment:
This is already the thirdApi url and the ip(apiUrl) is not being used anywhere. I keep it in case the ip started working again

Reply:
Ah, superstream wasnt working for me and it started working again as soon as I switched the url, but now the old one is working just fine, guess superstream is playing tricks on me